libs/aws/tests/integration_tests/chat_models/test_sagemaker_endpoint.py

```diff
@@ -0,0 +1,57 @@
+"""Test SageMakerEndpoint chat model."""
```

Comment:
ChatModels have a standard set of unit tests. Can you try adding them, and mark xfails/skip for the features that are not implemented. See tests for ChatBedrockConverse here.
https://github.com/langchain-ai/langchain-aws/blob/main/libs/aws/tests/unit_tests/chat_models/test_bedrock_converse.py#L33-L78